recipes/fastqpuri/meta.yaml

```diff
@@ -2,12 +2,12 @@ package:
   name: fastqpuri
   version: 1.0.5
 
-build:
-  number: 1
-
 source:
-  git_rev: v1.0.5
-  git_url: https://github.com/jengelmann/FastqPuri.git
+  url: https://github.com/jengelmann/FastqPuri/archive/v1.0.5.tar.gz
+  sha256: 33fedf39e6322428ff384a5815809f3721efe3fd93798807fdca8fbc50bd62c7
+
+build:
+  number: 2
```

Comment:
This should still stay as build number 0. As it will be the first time this package is built. You only need to update it if you change an already built package without changing the version number